src/lamp_core/standard_lamp.py

```diff
@@ -16,6 +17,7 @@ def __init__(self, name, base_color, shade_color):
         self.network = self.bluetooth.network
         self.bluetooth.enable()
         self.motion = MotionMPU6050(pin_sda=21, pin_scl=22)
+        self.temperature = TemperatureMPU6050(self.motion.accelerometer)
```

Comment:
the motion module broke for me on the esp32 we use, this might as well - i suspect just pins on the motion bit though. I commented it out for now locally